### ISSUE TITLE
Double output of an error if a not schema conform provider-metadata.json ist parsed

### ISSUE THREAD

**Issue by mfd2007** [csaf-poc/csaf_distribution#252]
"version": "0.9.4-46-g1b0e5b4",

If a provider-metadata.json with one problem in the schema validation, the problem is printed in twice in the report.

Example:

Add ```, "additional_field":"value"``` in provider-metadata.json

Error repot of checker:
```
       {
          "num": 7,
          "description": "provider-metadata.json",
          "messages": [
            {
              "type": 1,
              "text": "https://csaf-provider-1.csaf/.well-known/csaf/provider-metadata.json: Validating against JSON schema failed: \u003cnil\u003e"
            },
            {
              "type": 1,
              "text": "https://docs.oasis-open.org/csaf/csaf/v2.0/provider_json_schema.json#/additionalProperties: additionalProperties 'additional_field' not allowed"
            },
            {
              "type": 1,
              "text": "https://docs.oasis-open.org/csaf/csaf/v2.0/provider_json_schema.json#: doesn't validate with https://docs.oasis-open.org/csaf/csaf/v2.0/provider_json_schema.json#"
            },
            {
              "type": 1,
              "text": "https://csaf-provider-1/.well-known/csaf/provider-metadata.json: Validating against JSON schema failed: \u003cnil\u003e"
            },
            {
              "type": 1,
              "text": "https://docs.oasis-open.org/csaf/csaf/v2.0/provider_json_schema.json#/additionalProperties: additionalProperties 'additional_field' not allowed"
            },
            {
              "type": 1,
              "text": "https://docs.oasis-open.org/csaf/csaf/v2.0/provider_json_schema.json#: doesn't validate with https://docs.oasis-open.org/csaf/csaf/v2.0/provider_json_schema.json#"
            }
          ]
        },
```

**Comment by s-l-teichmann** [comment]
I would guess this stems from the fact that we unconditionally load the PMD from .wellknown and security.txt. Its is essential the same PMD so the reports are are generated from both.

A potential solution would add a checksum check with the already loaded one (which we have when we load the second one) and omit the validation when the sums are equal.

**Comment by s-l-teichmann** [comment]
PR #254 tries very hard to avoid duplicate checks and there reporting.

**Comment by Fadiabb** [comment]
Fixed with #245. Close here
